src/SFA.DAS.EmployerCommitmentsV2.Web/Models/Cohort/ApprenticeRequest.cs

```diff
@@ -2,10 +2,9 @@
 
 namespace SFA.DAS.EmployerCommitmentsV2.Web.Models.Cohort
 {
-    public class ApprenticeRequest : AssignRequest, IAuthorizationContextModel
+    public class ApprenticeRequest : AssignRequest
```

Comment:
Curious why IAuthorizaruibContextModel was removed?  But I see it;s been added to the base class IndexRequest

Reply:
All the requests inherit from one another and I needed to get the AccountLegalEntityId to be populated in one of the base classes so I just moved the IAuthorizationContextModel to the top of the hierarchy, If it works like I think it works.